src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -517,55 +523,56 @@ private AIState structureStep()
 
     private AIState decorationStep()
     {
-        if(!goToConstructionSite())
+        if (!goToConstructionSite())
         {
             return this.getState();
         }
-
-        if(job.getSchematic().doesSchematicBlockEqualWorldBlock() || job.getSchematic().getBlock().getMaterial().isSolid() /*|| job.getSchematic().getBlock() == Blocks.air*/)
+        //|| job.getSchematic().getBlock() == Blocks.air
+        if (job.getSchematic().doesSchematicBlockEqualWorldBlock()
+            || job.getSchematic().getBlock().getMaterial().isSolid())
         {
             //findNextBlock count was reached and we can ignore this block
             return findNextBlockNonSolid();
         }
 
         worker.faceBlock(job.getSchematic().getBlockPosition());
-        Block block = job.getSchematic().getBlock();
+        Block       block    = job.getSchematic().getBlock();
         IBlockState metadata = job.getSchematic().getBlockState();
 
         BlockPos coords = job.getSchematic().getBlockPosition();
-        int x = coords.getX();
-        int y = coords.getY();
-        int z = coords.getZ();
+        int      x      = coords.getX();
+        int      y      = coords.getY();
+        int      z      = coords.getZ();
 
-        Block worldBlock = world.getBlockState(coords).getBlock();
+        Block       worldBlock         = world.getBlockState(coords).getBlock();
         IBlockState worldBlockMetadata = world.getBlockState(coords);
-
-        if(block == null)//should never happen
+        //should never happen
+        if (block == null)
         {
             BlockPos local = job.getSchematic().getLocalPosition();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'local' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)